go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/apache/dubbo-go
 
 require (
+	github.com/dubbo/go-for-apache-dubbo v1.0.0
```

Comment:
why require github.com/dubbo/go-for-apache-dubbo?

Reply:
This should be left over from history. Trying to delete it will cause ci to fail

Reply:
fixed it.